platform/12-cosign-installer.sh

```diff
@@ -1,19 +0,0 @@
-#!/bin/bash
```

Comment:
I think @rgreinho had a particular use-case in mind for this script. Might want to get his opinion on moving the logic.